docker/dev/mongo/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM mongo:3.6.2-jessie
```

Comment:
I dont  see much value in having such files, there is no any extension

Reply:
Agree. Would be removed in next commits.